### DISCUSSION TITLE
lockwise

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#9] [Announcements]
# lockwise

Discover LOCKWISE, a unique meta-game experiment that transforms the zero-sum game 'Rock, Paper, Scissors' into a non-zero-sum puzzle. Explore this new twist on classic gameplay.

https://nonzerosum.games/lockwise.html

**Comment by Leo-McArle** [comment]
Super interesting! 
It reminds me of a video I watched recently about negotiation as a game mechanic [1]. Positional bargaining is interesting cause it can be a competitive positive-sum system, with two entities trying to get the maximum reward by providing for the other.
I wonder how this experiment would feel if the mechanic centered on getting the maximum possible satisfaction for all the entities as a collective.
I can't really think of a way to build that without a way bigger scope though, sort of like this market simulation [2].

[1]: https://www.youtube.com/watch?v=OKuhRoSVizs&ab_channel=MentalCheckpoint
[2]: https://www.youtube.com/watch?v=4SGkahKPgsQ&ab_channel=CoolAI

**Reply by jamesstephenbrown** [reply]
That definitely sounds like the sort of simulation I’m interested in making, and I’ve got a free half finished proof of concepts I could dust off, it would definitely get to the heart of non zero sum mess, I’m planning a blog about the market as a non zero system, you’ve inspired me to build a sim for that now. Thanks!

**Reply by jamesstephenbrown** [reply]
I wonder if this could make for a good evolutionary system. Where let's say you have a basic metric like "life points" and each shape in the system loses life slowly over time, in order to stay alive triangles need to touch a square, squares need to touch a circle, and circles need to touch a triangle. Then give them a genetic algorithm that learns behaviour - with the idea being that in order to survive the shapes would have to learn to both touch their goal shape but also allow themselves to be touched by the shape for whom they are the goal shape. Aligning the individual needs with the collective (this is what the next blog is about so this alignment is on my mind at the moment).
